### PR TITLE
Contribution instruction to README, --skip-missing-interpreters tip, and ditch Makefile

### PR DESCRIPTION
**Pull request by ArnaudBrousseau**
Switches from tox to tox2, and adds --skip-missing-interpreters (not everybody has pypy/pypy3 installed on their laptop.)

### CODE REVIEW

**Comment by ArnaudBrousseau** [comment]
@asottile shipit?

**Comment by asottile** [comment]
tox2 is a yelpism.  I'd instead suggest applying this just internally in the jenkins configuration.  Ping me in irc and I can show you a few examples where we do this

**Comment by ArnaudBrousseau** [comment]
Didn't know tox2 was a yelpism. Whoops. After discussion, removed the Makefile, it's not doing anything worth keeping (Travis just does "tox"). Added some contribution instructions to the README instead.

File: README.md [hunk 1]
```diff
@@ -53,6 +53,16 @@ together by a common "`wrapper`" node.
...
+# Contributing
+
+Use Github's pull request and issues system. To run tests:
+
+```
+tox --skip-missing-interpreters
+(use tox2 if you're running on Yelp's internal servers)
+```
+`--skip-missing-interpreters` will allow you to skip missing interpreters.
+Those will be tested by TravisCI when you issue a pull request.
```
Comment (asottile) [inline README.md]: nitpick: Travis CI

File: README.md [hunk 2]
```diff
@@ -53,6 +53,16 @@ together by a common "`wrapper`" node.
 the function's code object also has a name (`.func_code.co_name`) and that is
 what [cProfile][cprofile] looks at when it traces calls.
 
+# Contributing
+
+Use Github's pull request and issues system. To run tests:
+
+```
+tox --skip-missing-interpreters
+(use tox2 if you're running on Yelp's internal servers)
```
Comment (asottile) [inline README.md]: I wouldn't bother mentioning internal-specific instructions, shrugs

**Comment by ArnaudBrousseau** [comment]
boatit?

**Comment by asottile** [comment]
:horse::boat:s

**Pull request merged**